### ISSUE TITLE
There is no RGB in the file, attempting to read the RGB will throw a NullPointerException

### ISSUE THREAD

**Issue by Aynamic** [mreutegg/laszip4j#131]
If the file has no color, a NullPointerException occurs when trying to read the R G B

``` java
        LASReader lasReader = new LASReader(new File("D:\\110.las"));
        CloseablePointIterable closeablePoints = lasReader.getCloseablePoints();
        for (LASPoint lasPoint : closeablePoints) {
            char red = lasPoint.getRed();
        }
```

![图片](https://github.com/mreutegg/laszip4j/assets/24519941/3fc8a70d-0203-41a9-ab75-5a087a58afc7)

**Comment by mreutegg** [comment]
Thanks for reporting this issue. I updated LASPoint with a method to check for RGB data and throw an IllegalStateException when getRed(), getGreen() or getBlue() is called on a point without RGB data.

See https://github.com/mreutegg/laszip4j/commit/a9ce67eef5170c9b4be02b73875f27e6324a3399

**Comment by mreutegg** [comment]
Change is included in laszip4j 0.18 release.
